### PR TITLE
Add legal staycation terms (fixes #12332)

### LINKED ISSUE #12332: Add Terms and Service for upcoming contest
## Description

Terms & Service text: [doc](https://docs.google.com/document/d/1Jc6HtpvKI1bLv-pZkJUB4bSGDLNNowcYlm8WjsFqaI0/edit?usp=sharing)

URL: `/about/legal/terms/staycation`

Date: Needs to be live by 11/14. Can go live before this date, but team would prefer to not have it be too much earlier.

### PR DESCRIPTION
**Pull request by maureenlholland**
## One-line summary

Adds hard-coded legal page (content from https://docs.google.com/document/d/1Jc6HtpvKI1bLv-pZkJUB4bSGDLNNowcYlm8WjsFqaI0/edit, copy compared between doc and webpage with https://www.copyscape.com/compare.php, all matched correctly)

Must be live by **Nov. 14**

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/12332

## Testing

http://localhost:8000/en-US/about/legal/terms/staycation/

note: ~I have not added this to the legal index navigation, not sure it will be around long enough to merit that~ confirmed it is not needed in legal index navigation

### CODE REVIEW

**Comment by maureenlholland** [comment]
waiting on stakeholder review before merge, but it is unlikely changes will be requested from that review, so marking this as ready for code review

**Review by alexgibson — APPROVED** [review]
r+

**Pull request merged**
